.github/workflows/main.yml

```diff
@@ -8,5 +8,3 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run ansible-lint
         uses: ansible-community/ansible-lint-action@main
```

Comment:
Can someone check this change?

Reply:
The workflow reported as having run, and passed, so I think you're good on this change

Reply:
As for me, this change is totally fine if we really need `latest` so this will simply suppress all warnings related to this check.